avaframe/com1DFA/particleTools.py

```diff
@@ -214,7 +214,7 @@ def removePart(particles, mask, nRemove, reasonString='', glideSnow=0):
         particles dictionary
     """
     if reasonString != '':
-        log.info('removed %s particles %s' % (nRemove, reasonString))
+        log.debug('removed %s particles %s' % (nRemove, reasonString))
```

Comment:
I think this is also an important message, as sometimes there is a problem and particles are then also removed so it is a good hint that something could be wrong!

Reply:
Then we need to collect this information and make a summary log AFTER the simulation is done. Printing it directly during execution significantly slows down execution since it creates a bottleneck on the standard output, especially for parallel runs. I'll start an issue.